kv/memdb_buffer.go

```diff
@@ -25,6 +25,13 @@ import (
 	"github.com/pingcap/tidb/terror"
 )
 
+// Those limits is enforced to make sure the transaction can be well handled by TiKV.
+var (
+	EntrySizeLimit  = 8 * 1024 * 1024
```

Comment:
In this PR: https://github.com/pingcap/tidb/pull/2425/files,  requestMaxSize is 4 * 1024 * 1024.